t/rose-date/00-basic.t

```diff
@@ -300,4 +300,34 @@ file_cmp "$TEST_KEY.err" "$TEST_KEY.err" <<'__ERR__'
 [FAIL] [UNDEFINED ENVIRONMENT VARIABLE] ROSE_TASK_CYCLE_TIME
 __ERR__
 #-------------------------------------------------------------------------------
+# Test rose date --as-total runs without error
+TEST_KEY=$TEST_KEY_BASE-as-total
+run_pass "$TEST_KEY" rose date --as-total=s PT1M
+file_cmp "$TEST_KEY.err" "$TEST_KEY.err" </dev/null
+#-------------------------------------------------------------------------------
+# Test rose date --as-total=s P1DT1H1M1S
+TEST_KEY=$TEST_KEY_BASE-as-total-P1DT1H1M1S-s
+run_pass "$TEST_KEY" rose date --as-total=s P1DT1H1M1S
+file_cmp "$TEST_KEY.out" "$TEST_KEY.out" <<'__OUT__'
+90061.0
```

Comment:
Would make sense for converting to seconds certainly, for minutes and hours we would need to floor values though.